### PR TITLE
Setup multitenant algolia

### PR DESCRIPTION
**Pull request by vinnyjth**
### About

Sets up Multitenancy Algolia DAG to pull content items from Mutlitenant apps into a shared Multitenant Algolia instance. 

### Screenshots
 
image

image

### CODE REVIEW

File: dags/algolia_multitenant/algolia_content_dag.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    categories = []
+    category_names = Variable.get(
+        kwargs["client"] + "_apollos_postgres", deserialize_json=True, default_var=[]
+    )
+
+    active_filter = "(publish_at < NOW() or publish_at is null) and (expire_at > NOW() or expire_at is null)"
+
+    if len(category_names) > 0:
+        categories = pg_hook.get_records(
+            "select id from content_item_category where title in ('Go Do Good: Service Opportunities')"
```
Comment (redreceipt) [inline dags/algolia_multitenant/algolia_content_dag.py:41]: this isn't relevant for a global DAG

Comment (vinnyjth) [reply dags/algolia_multitenant/algolia_content_dag.py:41]: whoops, thanks

Comment (vinnyjth) [reply dags/algolia_multitenant/algolia_content_dag.py]: @redreceipt Fixed.

File: dags/algolia_multitenant/algolia_content_dag.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        keepalives_interval=10,
+        keepalives_count=5,
+    )
+
+    items = []
+    categories = []
+    category_names = map(
+        lambda r: f"'{r}'",
+        Variable.get(
+            kwargs["client"] + "_apollos_postgres",
```
Comment (redreceipt) [inline dags/algolia_multitenant/algolia_content_dag.py:36]: shouldn't we call this something else? `_indexed_categories` or something?

Comment (vinnyjth) [reply dags/algolia_multitenant/algolia_content_dag.py:36]: yah.... moving too fast. Will fix.

Comment (vinnyjth) [inline dags/algolia_multitenant/algolia_content_dag.py:36]:
```suggestion
            kwargs["client"] + "_indexed_category_names
```

**Comment by redreceipt** [comment]
for woodmen, wanna throw these category names in just to make sure it's working properly?

```
Next Steps
Events
Stories
Daily Scripture Reading
Weekly Community Group Guide
```

**Comment by vinnyjth** [comment]
> Next Steps
> Events
> Stories
> Daily Scripture Reading
> Weekly Community Group Guide

Yes! thank you, that's very helpful

**Comment by vinnyjth** [comment]
image


Matches the query directly in postgres: 
image

**Review by redreceipt — APPROVED** [review]

**Pull request merged**